airontools/constructors/layers.py

```diff
@@ -7,30 +7,30 @@
 from airontools.constructors.utils import regularizer
 
 
-def layer_constructor(
-    x,
-    name=None,
-    name_ext=None,
-    units=None,
-    num_heads=None,
-    key_dim=None,
-    value_dim=None,
-    activation=None,
-    use_bias=True,
-    sequential=False,
-    bidirectional=False,
-    return_sequences=False,
-    filters=None,
-    kernel_size=None,
-    padding="valid",
-    pooling=None,
-    pool_size=None,
-    conv_transpose=False,
-    strides=(1, 1),
-    sequential_axis=1,
-    advanced_reg=False,
-    **reg_kwargs
-):
+def layer_constructor(x,
+                      name=None,
+                      name_ext=None,
+                      units=None,
+                      num_heads=None,
+                      key_dim=None,
+                      value_dim=None,
+                      activation=None,
+                      use_bias=True,
+                      sequential=False,
+                      bidirectional=False,
+                      return_sequences=False,
+                      filters=None,
+                      kernel_size=None,
+                      padding='valid',
+                      pooling=None,
+                      pool_size=None,
+                      conv_transpose=False,
+                      strides=(1, 1),
+                      sequential_axis=1,
+                      advanced_reg=False,
+                      custom_model_obj=None,
```

Comment:
```suggestion
                      custom_model=None,
```
Please see below for more explanation